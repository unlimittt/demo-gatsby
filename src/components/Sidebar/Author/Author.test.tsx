import React from "react"
import Author from "./Author"
import Enzyme, { shallow } from "enzyme"
import Adapter from "enzyme-adapter-react-16"

Enzyme.configure({ adapter: new Adapter() })

describe("Author", () => {
  const props = {
    author: {
      name: "test",
      photo: "/photo.jpg",
      bio: "test",
    },
    isIndex: false,
  }

  it("renders correctly", () => {
    const wrapper = shallow(<Author {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
