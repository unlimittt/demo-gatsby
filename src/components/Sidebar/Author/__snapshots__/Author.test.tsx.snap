// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Author renders correctly 1`] = `
<div
  className="author"
>
  <mockConstructor
    to="/"
  >
    <img
      alt="test"
      className="author__photo"
      height="75"
      src="/photo.jpg"
      width="75"
    />
  </mockConstructor>
  <h2
    className="author__title"
  >
    <mockConstructor
      className="author__title-link"
      to="/"
    >
      test
    </mockConstructor>
  </h2>
  <p
    className="author__subtitle"
  >
    test
  </p>
</div>
`;
